ui/app/src/App.tsx

```diff
@@ -93,14 +92,31 @@ function App() {
       >
         <CreateStream
```

Comment:
maybe use `stream` is null to switch between `StreamStatus` and `CreateStream` components

---

This could eventually be on a `/{StreamManager-address}/create` sub-page, using a better routing structure like:

- `/{StreamManager-address}` - display all streams for a particular stream manager address (and load new ones as created), add filter params to filter down streams
- `/{StreamManager-address}/{creator-address}` - display all streams for a particular stream manager, by a particular creator (same as above, load new ones as they come in and add filter params)
- `/{StreamManager-address}/create` - Create a new Stream, re-route to detail view once created (using `registerStream` prop)
- `/{StreamManager-address}/{creator-address}/{stream-id}` - display the details of a particular stream

Reply:
linked to issue #53